data/110/849/986/1/1108499861.geojson

```diff
@@ -58,7 +64,7 @@
         "\u039a\u03bf\u03bc\u03b7\u03c4\u03b5\u03af\u03b1 \u03a4\u03c3\u03b9\u03b1\u03b3\u03af"
     ],
     "name:eng_x_preferred":[
-        "Chiayi County"
```

Comment:
Probably needs to be stored as an alt, no?

Reply:
Agree, I'll add a new commit...

Reply:
Actually these are a little hard for me to pull back out...

And generally these "County" values are now available in the `label:*` namespace so they are not removed completely, so I think it's OK?

For scope: In Taiwan this looks to affect 72 records.